docs/docs/building-a-blog.md

```diff
@@ -1,8 +1,7 @@
 ---
-title: Building a Blog
+title: ブログ構築
 ---
 
-This is a stub. Help our community expand it.
+このドキュメントは書きかけです。私たちのコミュニティーの発展を手伝ってください。
 
-Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
-Pull Request gets accepted.
+あなたの PullRequest が承認されるためにも[翻訳スタイルガイド](/contributing/gatsby-style-guide/) を一読してください。
```

Comment:
```suggestion
あなたの Pull Request が承認されるためにも[翻訳スタイルガイド](/contributing/gatsby-style-guide/) を一読してください。
```

Reply:
By the way, another file has the exact same problem. If you want to take it, you can take it, fix it, and open another PR.

Reply:
@smorimoto Okay will open another PR to fix the same issue.